README.md

```diff
@@ -91,6 +91,17 @@ app.UseRealTimeComm(yourSocketServiceLoaderInstance);
 
 Take a look at [library's SocketServiceLoader implementation](https://github.com/valudio/netsocket/blob/master/NetSocket/Sockets/SocketServiceLoader.cs) so you get a sense of what you could do.
 
+## Sending additional parameters to the server
+
+If you want to send custom data in the connection you can add in the url as a query string.
```

Comment:
If you want to send custom data through the socket connection you can add it to the url as a query string